.github/workflows/build.yaml

```
@@ -1,6 +1,8 @@
 # yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
 name: Build
-on: push
+on:
+  - push
+  - pull_request
```

Please use "proper" indentation in the future:

```suggestion
on:
- push
- pull_request
```

This doubly indented style is like:

```c
if (condition)
    {
        statement;
    }
```